keypom/keypom-airfoil
# Global config for supported wallets

# Description
currently there are different configs for create drops and claim pages. We should consolidate the wallet options into a single config so the supported wallets list can be easily maintained

But create token drop does not add wallets property in drop metadata so claim page will always show my near wallet now.

This PR only enables My Near Wallet 

Fixes #160 

# How to test
Show steps to replicate and test the feature/fixes in this PR

1. https://ericw-single-global-config.keypom-airfoil.pages.dev/drops
2. Create a token drop
3. You should be able to see in wallet options, there is only My Near Wallet
4. Attempt to claim a token drop and check that My Near Wallet is shown as the only option
5. Create an NFT
6. Attempt to claim an NFT just to check the wallet again
7. You can checkout locally and modify the config to see the changes


# Screenshots/Screen Recording of Implementation
Create Token Drop
<img width="589" alt="image" src="https://user-images.githubusercontent.com/19740800/221955122-a375b0b5-7b06-46bb-b012-11a51b13fa4a.png">
Create NFT Drop
<img width="479" alt="image" src="https://user-images.githubusercontent.com/19740800/221955584-6e70648b-c800-4464-8c4f-39346018db7e.png">

NEAR drop
<img width="634" alt="image" src="https://user-images.githubusercontent.com/19740800/221955200-96f29185-eb82-43f3-b791-269a1e953490.png">
NFT
<img width="623" alt="image" src="https://user-images.githubusercontent.com/19740800/221955361-1554c042-2a1b-46f5-bb26-01bd5034fbd1.png">
